css/index.css

```diff
@@ -1 +1,346 @@
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 80%;
+    margin: auto;
+  }
+}
+.dot-ln .nav-bar nav {
+  width: 45%;
+  display: flex;
+  justify-content: space-around;
+}
+@media (max-width: 500px) {
+  .dot-ln .nav-bar nav {
+    margin-top: 20px;
+    margin-bottom: 10px;
+    width: 100%;
+  }
+}
+.dot-ln .nav-bar nav a {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+.footer {
+  width: 100%;
+  border-top: 2px dashed silver;
+  background: #FFEBCD;
+}
+.footer p {
+  text-align: center;
+  color: #212529;
+  font-size: 1.6rem;
+  padding: 20px;
+}
+.home {
+  margin: auto;
+  width: 100%;
+  color: #212529;
+  font-weight: bold;
+  overflow: visible;
+}
+.home header {
+  padding-bottom: 10px;
+  border-bottom: 2px dashed #C0C0C0;
+  width: 100%;
+  margin-top: 0;
+}
+@media (max-width: 500px) {
+  .home header {
+    width: 80%;
+    margin: auto;
+  }
+}
+.home header img {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+.home .content-section {
+  margin: 30px 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+@media (mobile) {
+  .home .content-section {
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: center;
+  }
+}
+.home .content-section .text-content {
+  width: 48%;
+  padding-right: 1%;
+}
+@media (max-width: 500px) {
+  .home .content-section .text-content {
+    width: 80%;
+    margin: auto;
+  }
+}
+.home .content-section .img-content {
+  width: 48%;
+  padding-left: 1%;
+}
+@media (max-width: 500px) {
+  .home .content-section .img-content {
+    width: 80%;
+    margin: auto;
+  }
+}
+.home .content-section .img-content img {
+  border-radius: 10px;
+}
+.home .inverse-content {
+  padding-bottom: 30px;
+  border-bottom: 2px dashed #C0C0C0;
+}
+@media (mobile) {
+  .home .inverse-content {
+    flex-direction: column;
+  }
+}
+.home .inverse-content .text-content {
+  padding-left: 1%;
+  padding-right: 0;
+}
+@media (max-width: 500px) {
+  .home .inverse-content .text-content {
+    order: 1;
+  }
+}
+.home .inverse-content .img-content {
+  padding-right: 1%;
+  padding-left: 0;
+}
+@media (max-width: 500px) {
+  .home .inverse-content .img-content {
+    order: 2;
+  }
+}
+.home .content-destination {
+  width: 75%;
+  margin: 0 auto 30px;
+}
+.home .content-destination img {
+  border-radius: 10px;
+}
+.home .content-pick {
+  padding-top: 30px;
+  border-top: 2px dashed #C0C0C0;
+  display: flex;
+  justify-content: space-between;
+}
+@media (max-width: 500px) {
+  .home .content-pick {
+    flex-direction: column;
+  }
+}
+.home .content-pick .destination {
+  width: 30%;
+  margin-bottom: 30px;
+}
+@media (max-width: 500px) {
+  .home .content-pick .destination {
+    width: 80%;
+    margin: auto;
+    margin-bottom: 20px;
+  }
+}
+.home .content-pick .destination button {
+  width: 75%;
+  height: height;
+  background: #17A2B8;
+  color: #FFFFFF;
+  border-radius: 15px;
+  font-size: 1.75rem;
+  font-weight: bold;
```

Comment:
add a padding here so the buttons can be big bigger for mobile, like the design file.  Mobile views always have thicker buttons to make it easy to use the finger to click.